ROADMAP.md

```diff
@@ -0,0 +1,22 @@
+# Our Roadmap
+
+Hey, there. Thanks so much for the interest in openSNP and what we're trying to accomplish over the next weeks and months.
+
+We have some ongoing efforts which need to be done as well as some bug fixes and new features we'd love to implement. If you think we should focus on something else: Please just comment, add it or talk to us, whatever you feel like doing!
+
+
+## Ongoing Efforts
```

Comment:
🎉👏 I love this!
On Thu, Mar 3, 2016 at 6:19 PM Bastian Greshake <notifications@github.com>
wrote:

> In ROADMAP.md
> <https://github.com/openSNP/snpr/pull/265#discussion_r54965859>:
>
> > @@ -0,0 +1,22 @@
> > +# Our Roadmap
> > +
> > +Hey, there. Thanks so much for the interest in openSNP and what we're trying to accomplish over the next weeks and months.
> > +
> > +We have some ongoing efforts which need to be done as well as some bug fixes and new features we'd love to implement. If you think we should focus on something else: Please just comment, add it or talk to us, whatever you feel like doing!
> > +
> > +
> > +## Ongoing Efforts
>
> ok, see the next version [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openSNP/snpr/pull/265/files#r54965859>.
>